book/docs/setup/MacOS.md

```diff
@@ -182,7 +180,7 @@ Segmentation fault: 11
 ```
 
```

Comment:
Running either the server or client will require `GC_DONT_GC=1` be prepended to the command.

This is because the garbage collector incorrectly marks some of the dependencies as unneeded. See [https://github.com/renzwo/cardano-plutus-apps-install-m1/blob/main/README.md](https://github.com/renzwo/cardano-plutus-apps-install-m1/blob/main/README.md)